tools/docker/start.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
+set -ex
 
 cd web-platform-tests
-git pull --depth=50
+git fetch origin -n -q --depth=50
+git checkout -b master origin/master
```

Comment:
Is there a separate step that checks out the right revision?

Reply:
Yeah, although I have a followup to do this more obviously.